crates/sui-indexer/README.md

```diff
@@ -13,7 +13,8 @@ export LDFLAGS="-L/opt/homebrew/opt/postgresql@15/lib"
 export CPPFLAGS="-I/opt/homebrew/opt/postgresql@15/include"
 export PATH="/opt/homebrew/opt/postgresql@15/bin:$PATH"
 ```
-- make sure you have libpq installed: `brew install libpq`, and in your profile, add `export PATH="/opt/homebrew/opt/libpq/bin:$PATH"`. If this doesn't work, try `brew link --force libpq`.
+- make sure you have libpq installed: `brew install libpq`, and in your profile, add `export PATH="/opt/homebrew/opt/libpq/bin:$PATH"`. If this doesn't work, try `brew link --force libpq`. 
+    - Note: Make sure that the `libpq` bin occurs after the `postgres` bin in your `PATH` variable. Otherwise you will get `initdb` errors.
```

Comment:
Added this note as I had issues running tests involving postgres because my `PATH` had the correct elements, but in the wrong order, so it was using the wrong `initdb` in `libpq` as opposed to the installed postgresql@15

Reply:
I can? But it's so small...